### ISSUE TITLE
classify_bins blast nt database errors

### ISSUE THREAD

**Issue by zhibaosuo** [bxlab/metaWRAP#430]
I run classify_bins using MetaWRAP v=1.3.2, blast+ 2.13.0, and the error as follows:

The file /systemRoot/db/NCBI_nt/nt.00.nhd doesnt exist, which likely means that you  havent set the correct path to your NCBI_nt database or havent downloaded it.

The nt database was downloaded: wget ftp://ftp.ncbi.nlm.nih.gov/blast/db/nt.*.tar.gz

**Comment by kalonji08** [comment]
Hi have you tried adding the nt database path to the config file?
Try:
which config-metawrap

then do:
gedit /PATH/TO/WHICHCONFIG/FILE 

there you can add the actual PATH of you databases
Hopefully it works

**Comment by xubiy** [comment]
I have also encountered this issue, but my database configuration is correct, and after decompressing the file, I couldn't find nt.00.nhd either.

**Comment by sprdmt** [comment]
Could it be that NCBI changed the namind pattern of the db files? I also can't find the nt.00.nhd, but there's a nt.000.nhd in the folder. 


-----

Okay, I found a solution that works for me.
First, I changed the file conda/envs/metawrap-2/bin/metawrap-modules/classify_bins.sh : change lines 79-80 to
if [ ! -f "${BLASTDB}/nt.000.nhd" ]; then
	error "The file ${BLASTDB}/nt.000.nhd doesnt exist, which likely means that you havent set the correct path to your NCBI_nt database or\

Basically I just changed the nt.00.nhd to nt.000.nhd 

After that I used an advice _originally posted by @lixiang117423 in https://github.com/bxlab/metaWRAP/issues/266#issuecomment-1593045519_
so I removed the file blastn from conda/envs/metawrap-2/bin/ and then soft-linked the same file from another environment with freshly installed BLAST:
ln -s conda/envs/blast-env/bin/blastn conda/envs/metawrap-2/bin/blastn

Hope this helps!
